flask_multipass/core.py

```diff
@@ -528,6 +535,8 @@ def handle_login_form(self, provider, data):
         try:
             response = provider.process_local_login(data)
         except MultipassException as e:
+            if isinstance(e, NoSuchUser) and current_app.config['MULTIPASS_HIDE_NO_SUCH_USER']:
+                e = InvalidCredentials(e.provider)
```

Comment:
As far as I understand, yes. The only way to preserve the traceback that I know of is with `raise ... from ...`. This is not relevant at the moment, since we don't let the exception bubble up and, instead, we pass it to `handle_auth_error()` (which only uses the exception message).